amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -198,7 +199,18 @@ router.get('/:startTimestamp', async (req, res) => {
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   let startTimestamp = req.params.startTimestamp;
+   let body = {
+      startTimestamp: parseInt(startTimestamp),
+      ...req.body
+   }
+   try {
+     const shift = await putShift(body);
+     res.send(shift)
+   }
+   catch (err) {
+      res.status(404).json({ error });
```

Comment:
The status isn't always 404, just send the error in the status

Reply:
Sounds good. I also changed the docstring from 404 to 400.